components/counter/Counter.js

```diff
@@ -6,28 +6,32 @@ import theme from './theme.css';
 
 class Counter extends PureComponent {
   static propTypes = {
+    borderColor: PropTypes.oneOf(['neutral', 'mint', 'aqua', 'violet', 'teal', 'gold', 'ruby', 'tealdark']),
     className: PropTypes.string,
     color: PropTypes.oneOf(['neutral', 'mint', 'aqua', 'violet', 'teal', 'gold', 'ruby']),
     count: PropTypes.number,
-    dark: PropTypes.bool,
+    inactive: PropTypes.bool,
     maxCount: PropTypes.number,
     size: PropTypes.oneOf(['small', 'medium']),
   };
 
   static defaultProps = {
+    borderColor: 'neutral',
     color: 'neutral',
+    inactive: false,
     size: 'medium',
   };
 
   render() {
-    const { className, color, count, dark, maxCount, size, ...others } = this.props;
+    const { borderColor, className, color, count, inactive, maxCount, size, ...others } = this.props;
 
     const classes = cx(
       theme.counter,
       theme[color],
       theme[size],
+      theme[`border-${borderColor}`],
       {
-        [theme.dark]: dark,
+        [theme.inactive]: inactive,
```

Comment:
**B:** PHPStorm doesn't allow clicking through to this class definition. Can you use array notation here? Makes it easier to refactor too, if necessary.

Reply:
Agreed @nickwaelkens! I'll do it in a separate PR.

Reply:
I guess you mean `theme['inactive']`? That does give me the ability to click through, but to the wrong things...
Also, I don't think we should start changing things because PhpStorm doesn't support it.
![screen shot 2017-10-17 at 11 02 58](https://user-images.githubusercontent.com/330765/31656708-be43887a-b32c-11e7-8b13-2a26c19a8e6a.png)

Reply:
**edit:** I did it in the smae PR 😉